packages/neuron-ui/src/types/Controller/index.d.ts

```diff
@@ -311,4 +311,16 @@ declare namespace Controller {
     }
     type Response = boolean
   }
+
+  namespace GetSUDTTokenInfo {
+    interface Params {
+      tokenId: string
+    }
+    interface Response {
+      tokenId: string
+      symbol: string
+      tokenName: string
+      decimal: string
+    }
```

Comment:
Would it better to simplify the following properties?
`tokenId` => `id`
`tokenName` => `name`

Reply:
tokenId and tokenName are also used in other code, so keep consistency maybe better